packages/utils-json/src/schema-validator.ts

```diff
@@ -319,6 +303,22 @@ const prettify = (errors: ajv.ErrorObject[]) => {
 };
 
 export const validate = (schema: object, json: object, getLocation?: JSONLocationFunction): SchemaValidationResult => {
+    /*
+     * If we want to use the ajv types in TypeScript, we need to import
+     * ajv in a lowsercase variable 'ajv', otherwise, we can't use types
+     * like `ajv.Ajv'.
+     */
+    const validator = new ajv.default({ // eslint-disable-line new-cap
+        $data: true,
+        allErrors: true,
+        logger: false,
+        useDefaults: true,
+        verbose: true
+    } as ajv.Options);
+
+    addFormats(validator);
+    validator.addKeyword('regexp');
+
```

Comment:
Is there a reason we can no-longer reuse a single validator instance across calls?

Reply:
ajv complains if you try to analyze more than one schema with the same id and we do that during tests :)